Timer/js/index.js

```diff
@@ -0,0 +1,187 @@
+/**
+ * clear object value
+ * @param {object} element
+ */
+function clear(element) {
+  element.value = '';
+}
+
+/**
+ * disable object
+ * @param {object} element
+ */
+function disable(element) {
+  element.setAttribute('disabled', 'disabled');
+}
+
+/**
+ * enable object
+ * @param {object} element
+ */
+function enabled(element) {
+  element.removeAttribute('disabled');
+}
+
+/**
+ * loop function from parameter
+ * @param {string} func
+ */
+function loop(func) {
+  for (let key in window.get) {
+
+    if (window.get.hasOwnProperty(key)) {
+      window[func](window.get[key]);
+    }
+
+  }
+}
+
+/**
+ * get seconds count
+ * @returns {number}
+ */
+function getTime() {
+  let time = 0;
+  let toSeconds = 3600;
+
+  for (let key in window.get) {
+
+    if (window.get.hasOwnProperty(key)) {
+      time += (parseInt(window.get[key].value) * toSeconds);
+      toSeconds /= 60;
+    }
+
+  }
+
+  return time;
+}
+
+/**
+ * add zeros to result
+ */
+function addZero() {
+  for (let key in window.get) {
+
+    if (window.get.hasOwnProperty(key)) {
+
+      if (window.get[key].value === '') {
+        window.get[key].value = '00';
+      }
+
+      if (window.get[key].value < 10 && window.get[key].value !== '00') {
+        window.get[key].value = `0${ window.get[key].value}`;
+      }
+
+    }
+
+  }
+}
+
+/**
+ * stop timer
+ */
+function stop() {
+  window.clearInterval(window.timerId);
```

Comment:
you don't need to call clearInterval from window object. Only call clearInterval()